images/scripts/fedora-36.install

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
```

Comment:
https://github.com/cockpit-project/cockpit/pull/16935 just landed, so we don't need the `*.install` file any more now. However, if you don't need to re-push, it's fine -- I'll send a PR to drop them wholesale anyway.

Reply:
Ok